packages/flutter/test/widgets/layout_builder_test.dart

```diff
@@ -663,6 +663,38 @@ void main() {
     expect(spy.performLayoutCount, 3);
     expect(spy.performResizeCount, 2);
   });
+
+  testWidgets('LayoutBuilder descendant widget can access [RenderBox.size] when rebuilding during layout', (WidgetTester tester) async {
+    Size? childSize;
+    int buildCount = 0;
+
+    Future<void> pumpTestWidget(Size size) async {
+      await tester.pumpWidget(
+        // Center is used to give the SizedBox the power to determine constraints for LayoutBuilder
+        Center(
+          child: SizedBox.fromSize(
+            size: size,
+            child: LayoutBuilder(builder: (BuildContext context, BoxConstraints constraints) {
+              buildCount++;
+              if (buildCount > 1) {
+                final _RenderLayoutSpy spy = tester.renderObject(find.byType(_LayoutSpy));
+                childSize = spy.size;
```

Comment:
what happen if we try to access the size in the first build? we should make sure this situation will throw.

Reply:
The RO is null when first build and we can not access size by it.